js/controllers/hotkeys.js

```diff
@@ -104,14 +110,3 @@ HotkeysController.prototype.onKeydown_ = function(e) {
     }
   }
 };
-
-HotkeysController.prototype.onMouseWheel_ = function(e) {
```

Comment:
is this the code to fix the issue with the mouse zoom? didn't this already land? does this need a rebase?

Reply:
I think i need a rebase! Will fix :)